deploy/6.8.StrategydYdXSoloMargin.js

```diff
@@ -0,0 +1,80 @@
+module.exports = async ({ getNamedAccounts, deployments, getChainId }) => {
+    const { deploy, execute } = deployments;
+    let { DAI, USDC, WETH, deployer, unirouter, dYdXSoloMargin } = await getNamedAccounts();
+    const chainId = await getChainId();
+    const controller = await deployments.get('StrategyControllerV2');
+    const vaultManager = await deployments.get('yAxisMetaVaultManager');
+    const Converter = await deployments.get('StableSwap3PoolConverter');
+
+    if (chainId != '1') {
+        const dai = await deployments.get('DAI');
+        DAI = dai.address;
+        const usdc = await deployments.get('USDC');
+        USDC = usdc.address;
+        const weth = await deployments.get('WETH');
+        WETH = weth.address;
+        const router = await deployments.get('MockUniswapRouter');
+        unirouter = router.address;
+        await deploy('dYdXSoloMargin', {
+            from: deployer,
+            contract: 'MockdYdXSoloMargin',
+            log: true,
+            args: [
+                [2, 3],
+                [USDC, DAI]
+            ]
+        });
+        const deployeddYdXSoloMargin = await deployments.get('dYdXSoloMargin');
+        dYdXSoloMargin = deployeddYdXSoloMargin.address;
+    }
+
+    const deployedStrategyUSDC = await deploy('StrategydYdXSoloMarginUSDC', {
+        from: deployer,
+        contract: 'StrategydYdXSoloMargin',
+        log: true,
+        args: [
+            dYdXSoloMargin,
+            2,
```

Comment:
Do they have it documented somewhere which IDs correspond to which tokens? If so, it'd be good to link to that documentation (or replicate it) here.

Reply:
Found it: https://docs.dydx.exchange/#solo-markets
Let's just link to that here and in the strategy itself.